### DISCUSSION TITLE
No automatic updates offered

### DISCUSSION THREAD

**Discussion by HorstBrack75** [osy86/HaC-Mini#977] [Q&A] [answered]
I don’t get a notification for the 12.2. update. I already noticed the same thing for 12.1. which I also had to manually download from the AppStore since it was never offered in the update pane. 

I am running 12.1 on 3.1.4 on a NUC8i8HVK. Everything else is working great.

Did you guys get notifications for 12.1 and 12.2?

Am holding it wrong?

**Comment by MokOopsing** [comment]
not yet

**Comment by osy** [comment]
I see it fine

<img width="780" alt="image" src="https://user-images.githubusercontent.com/50960678/151905387-ac0ddd7d-c28d-4d73-8030-7e4db43db6f0.png">

Make sure you're not in the beta update channel: https://beta.apple.com/sp/betaprogram/unenroll

**Reply by HorstBrack75** [reply]
<img width="668" alt="Screenshot 2022-02-01 at 09 42 28" src="https://user-images.githubusercontent.com/2854956/151937399-e7a99bc7-a231-4651-8f35-a1efa2b4af89.png">

Unfortunately I don't see it. I am not in the beta update channel and sip is on. Any other ideas?

**Reply by Maijestro** [reply]
I have the same problem.I am not in the beta update channel and sip is on.

**Reply by Maijestro** [reply]
I have the same problem.I am not in the beta update channel and sip is on. My last installed version is v3.1.4 on NUC8i8HVK.

**Comment by Maijestro** [comment]
I think it's the HacMini 3.1.4 Installer. It does not patch the Monterey Installer. This selection has a gray background and cannot be selected. I reinstalled Monterey 12.2 with HacMini 3.4.

<img width="605" alt="Bildschirmfoto 2022-02-03 um 17 47 46" src="https://user-images.githubusercontent.com/94802864/152388757-f0dac9db-e61a-4965-812e-5217f1515e31.png">


<img width="766" alt="Bildschirmfoto 2022-02-03 um 17 51 55" src="https://user-images.githubusercontent.com/94802864/152389370-4f1bb97a-a056-4a18-aab8-b6eb8256d33b.png">



<img width="424" alt="Bildschirmfoto 2022-02-03 um 17 38 52" src="https://user-images.githubusercontent.com/94802864/152389492-29dd369a-e19c-4508-a4cb-7851198652ed.png">

**Reply by osy** [reply]
That's unrelated. Monterey installer doesn't support patching.

**Reply by osy** [reply]
Your screenshot shows two empty boxes for update channel and source. That's a problem, those boxes shouldn't appear if you're on release channel. My guess is that you've signed up for beta or something and reverted it and it corrupted the settings.

**Reply by osy** [reply]
Run the following and see if you still see the empty boxes

```
$ sudo /System/Library/PrivateFrameworks/Seeding.framework/Versions/A/Resources/seedutil enroll DeveloperSeed
$ sudo /System/Library/PrivateFrameworks/Seeding.framework/Versions/A/Resources/seedutil unenroll
```

**Comment by Maijestro** [comment]
It worked. Thanks so much ,-) 


<img width="398" alt="Bildschirmfoto 2022-02-03 um 18 44 15" src="https://user-images.githubusercontent.com/94802864/152399195-72909620-43a8-45a4-adbd-68cc17d485bd.png">

**Comment by osy** [comment, marked as answer]
Reposting my comment as the answer:

Run the following to reset your update source.

```
$ sudo /System/Library/PrivateFrameworks/Seeding.framework/Versions/A/Resources/seedutil enroll DeveloperSeed
$ sudo /System/Library/PrivateFrameworks/Seeding.framework/Versions/A/Resources/seedutil unenroll
```

**Reply by osy** [reply]
FWIW this is not a HaC mini bug but a macOS bug since this issue often shows up on Reddit.

**Reply by MokOopsing** [reply]
Still no notification for the 12.2, any good suggestions?

**Reply by osy** [reply]
The above doesn't work for you?

**Reply by MokOopsing** [reply]
> The above doesn't work for you?

yes, I wander if I should download 12.2 from appstore and manully install it

**Reply by HorstBrack75** [reply]
Did not work for me neither. I now downloaded 12.2 from the Appstore and re-installed.

**Comment by Maijestro** [comment]
I'm on Monterey 12.2 but only with a fresh install. I don’t get a notification for the 12.2.1 update.
I have read that it is a problem with the csr-active-config. Does HacMini 3.1.4 use this config?

**Reply by osy** [reply]
No

**Reply by Maijestro** [reply]
Do you have any ideas? I would test it....it's not a problem for me. HacMini 3.1.4 boots from external USB.

**Reply by osy** [reply]
Maybe try enabling the "boot security" option from the HaC mini installer? I don't know if that will make a difference, but I have it on and it works fine.

**Reply by MokOopsing** [reply]
@osy the other issue, I checked this option, but still show AMD gpu, did this meet expectations? if not, how could I make the my the option take effect?
<img width="479" alt="image" src="https://user-images.githubusercontent.com/39441028/153695370-c131c9cc-f1a0-470a-9b4a-eb582f215dc5.png">

<img width="586" alt="image" src="https://user-images.githubusercontent.com/39441028/153695378-d2a039f8-8c52-40c8-9ccb-def2252c2760.png">

**Reply by osy** [reply]
That item is missing a translation. It sets the codec used for video decoding only. It has nothing to do with the GPU.

**Reply by MokOopsing** [reply]
> That item is missing a translation. It sets the codec used for video decoding only. It has nothing to do with the GPU.

ok i see

**Reply by Maijestro** [reply]
> Maybe try enabling the "boot security" option from the HaC mini installer? I don't know if that will make a difference, but I have it on and it works fine.

With this Option „boot Security“ on HacMini 3.1.4 no boot and the nuc is going to Mac recovery Modus.

**Reply by osy** [reply]
You have to follow the instructions to boot with security. Don't worry about it if it's too complicated. It was just an idea.

**Reply by Maijestro** [reply]
>You have to follow the instructions to boot with security. Don't worry about it if it's too complicated. It was just an idea.

Can you upload a EFI folder for me.? I can test it with your settings.

**Reply by HorstBrack75** [reply]
I can confirm that enabling boot security solved the issue. I also re-installed 12.1 and 12.2 manually since no updates were offered. On 12.2 I did not get the 12.2.1 offered.

Now I activated boot security and the update instantly appears.

**Reply by Maijestro** [reply]
> I can confirm that enabling boot security solved the issue. I also re-installed 12.1 and 12.2 manually since no updates were offered. On 12.2 I did not get the 12.2.1 offered.
> 
> Now I activated boot security and the update instantly appears.

How did you enable boot security ?

**Reply by HorstBrack75** [reply]
I re-installed HaC Mini 3.1.4 with the boot security option. 

It then boots to recovery and you have to follow the instructions here:

https://github.com/osy/HaC-Mini/blob/master/post-installation/support.md#cannot-boot-when-secure-boot-is-enabled

**Reply by Maijestro** [reply]
It works "boot security" was the problem. Many thanks for help

<img width="761" alt="Bildschirmfoto 2022-02-12 um 19 33 54" src="https://user-images.githubusercontent.com/94802864/153723800-62eed649-cb23-43ac-81eb-968b1e380258.png">
.
